aten/src/ATen/ParallelNative.cpp

```diff
@@ -277,8 +278,8 @@ std::shared_ptr<c10::ivalue::Future> intraop_launch_future(
   }
   return future;
 #else
-  // TODO: caffe2::ThreadPool doesn't support submitting tasks separately and
-  // running in parallel. Should fix it when this API becomes popular.
+  // TODO: caffe2::PThreadPool only provides a data-parallel API.
+  // Task parallelism is not currently supported.
```

Comment:
This is mainly used in `at::launch`. I dont think we are there yet to support it on mobile.

Reply:
And only fork/wait ops use this.

Reply:
Thanks.